lib/model/map/UserTableMap.php

```diff
@@ -51,12 +51,12 @@ public function initialize()
 	public function buildRelations()
 	{
     $this->addRelation('actor', 'actor', RelationMap::MANY_TO_ONE, array('id' => 'id', ), 'CASCADE', null);
-    $this->addRelation('auditLog', 'auditLog', RelationMap::ONE_TO_MANY, array('id' => 'user_id', ), 'SET NULL', null);
```

Comment:
(Same as previous comment.)

Reply:
Fixed.

Reply:
Based on our discussion below, I have reverted my fix to re-order these items.